packages/blocks/src/store/process-block-type.js

```diff
@@ -110,6 +110,14 @@ export const processBlockType =
 			null
 		);
 
+		if ( settings.apiVersion <= 2 ) {
+			warning(
+				`The block "${ name }" is registered with API version 2 or lower. This means that the post editor may be rendered outside the iframe.\n` +
```

Comment:
> This means that the post editor may be rendered outside the iframe.

I think we need to adjust the wording here. I read it as there's an iframe, but the post editor renders outside of it. IIRC, this isn't true. With lower API versions, the post editor is rendered without using an iframe.

Reply:
3e1e1d22b0c9fc361decbcd133620bbe5d5ba95d

> This means that the post editor may work as a non-iframe editor.

What do you think of the updated message above? Other suggestions are welcome 😄